changelog.d/pa-2477.changed

```diff
@@ -0,0 +1,15 @@
+taint-mode: Historically, the matching of taint sinks has been somewhat imprecise.
+For example, `sink(ok if tainted else ok)` was flagged. Recently, we made sink-
```

Comment:
and it should not be flagged because taint in a condition does not matter, because the result will always
be 'ok' ?

Reply:
Correct.